dallinger/config.py

```diff
@@ -182,6 +182,12 @@ def override(self, *args, **kwargs):
         self.data.popleft()
 
     def get(self, key, default=marker):
+        if key == "auto_recruit":
```

Comment:
Are you planning to make this generic so that other variables can be manipulated this way?

Reply:
For now no.

Reply:
Cool makes sense. In the meantime can you leave a comment at the top of this region explaining briefly what happens (i.e. the variable is taken from Redis if it exists), and noting that in the future this logic might be generalized to other variables?

Reply:
Done